src/ansys/dpf/composites/ply_wise_data.py

```diff
@@ -0,0 +1,87 @@
+"""Methods to get ply-wise data from a result field."""
+
+from enum import Enum, IntEnum
+from typing import Union
+
+from ansys.dpf.core import Field, MeshedRegion, Operator, operators
+from ansys.dpf.gate.common import locations
+
+__all__ = ("ReductionStrategy", "get_ply_wise_data")
+
+
+class ReductionStrategy(Enum):
+    """The reduction strategy to get from spot values (BOT, MID, TOP) to a single value."""
```

Comment:
```suggestion
    """Provides the reduction strategy for getting from spot values (BOT, MID, TOP) to a single value."""
```
A description for a class should always being in a verb that ends in "s" or "es" so that summary tables are consistent in format.